src/http/ngx_http_mruby_ssl.c

```diff
@@ -151,6 +173,8 @@ void ngx_mrb_ssl_class_init(mrb_state *mrb, struct RClass *class)
   mrb_define_method(mrb, class_ssl, "certificate_key=", ngx_mrb_ssl_set_cert_key, MRB_ARGS_REQ(1));
   mrb_define_method(mrb, class_ssl, "certificate_data=", ngx_mrb_ssl_set_cert_data, MRB_ARGS_REQ(1));
   mrb_define_method(mrb, class_ssl, "certificate_key_data=", ngx_mrb_ssl_set_cert_key_data, MRB_ARGS_REQ(1));
+  mrb_define_class_method(mrb, class_ssl, "accept_client", ngx_mrb_ssl_accept_client, MRB_ARGS_NONE());
+  mrb_define_class_method(mrb, class_ssl, "reject_client", ngx_mrb_ssl_reject_client, MRB_ARGS_NONE());
```

Comment:
Are there any reasons that accept_client and reject_client must be class method?

Reply:
there are only one state that be passed to SSL context . so it is needless to be able to access by mutiple object.

Reply:
What's the difference between {accept,reject}_client and other SSL methods?
All they work with a single state (ngx_http_mruby_srv_conf_t). Both look same to me.

Reply:
IMO, I prefer that other SSL methods are also defined as class method.
I guess the ability to aceess single state by mutiple object is confusing newcommer.

I would like to define  `Nginx::SSL` methods which is added newly as class method.

Reply:
@matsumotory Any comments?

I'm OK with it for now. We can discuss the topic later.

Reply:
I think it should be an instance method. If the passing of data by the instance variable does not occur obviously, we recommend a class method. However, I think that it should be an instance method when considering future extensibility and semantics since an SSL instance can be regarded as the SSL object corresponding to a specific request. It is relatively easy to switch from an instance method to a class method.

Reply:
I understand why it should be an instance method. I will fix it.